### DISCUSSION TITLE
Corfu popup text not visible in PuTTY

### DISCUSSION THREAD

**Discussion by LemonBreezes** [minad/corfu#485] [Q&A]
Hi. Basically, when I change the default face in the Corfu popup from `corfu-default` to `default` in `corfu--popup-show` from `corfu-terminal`, the Corfu popup looks really legible, like on the right side of this screenshot:
![image](https://github.com/user-attachments/assets/616285c0-1349-4899-b373-21c9f4bbfdf2)
However, when I use the function as it is by default, the popup looks like this:
![image](https://github.com/user-attachments/assets/0e0e7621-0150-4c80-a2cf-eebab53cc4b1)

**Comment by LemonBreezes** [comment]
Just added this to my TTY config:
![image](https://github.com/user-attachments/assets/be324fdd-6e1b-4a36-8cf4-62e5e231a66c)

**Comment by minad** [comment]
Hi! This seems to be an issue of the theme or of the `corfu-default` face.

~~~ elisp
  '((((class color) (min-colors 88) (background dark)) :background "#191a1b")
    (((class color) (min-colors 88) (background light)) :background "#f0f0f0")
    (t :background "gray"))
~~~

My assumption is that the `class` and `min-colors` specification is too strong and we also have to provide a weaker definition, which also applies to terminals with fewer colors.
